tempto-core/src/main/java/com/teradata/tempto/fulfillment/table/RepositoryTableDefinition.java

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
I don't understand why do you move this class to runner.

If you need to create modules for hive and jdbc, then I could imagine that hive and jdbc are using core, and runner is using core, hive and jdbc. Why do you put it here?